packages/polaris-viz/src/components/LegendContainer/components/HiddenLegendTooltip.tsx

```diff
@@ -0,0 +1,173 @@
+import {
+  Fragment,
+  useCallback,
+  useEffect,
+  useMemo,
+  useRef,
+  useState,
+} from 'react';
+import type {ReactNode} from 'react';
+import {createPortal} from 'react-dom';
+import {
+  changeColorOpacity,
+  useChartContext,
+  useTheme,
+} from '@shopify/polaris-viz-core';
+import type {BoundingRect} from '@shopify/polaris-viz-core';
+
+import type {LegendData} from '../../../types';
+import {TOOLTIP_BG_OPACITY} from '../../../constants';
+import {useBrowserCheck} from '../../../hooks/useBrowserCheck';
+import {useRootContainer} from '../../../hooks/useRootContainer';
+import {useColorVisionEvents} from '../../../hooks/ColorVisionA11y';
+import {TOOLTIP_MARGIN} from '../../TooltipWrapper';
+import {Legend} from '../../Legend';
+
+import style from './HiddenLegendTooltip.scss';
+
+interface Props {
+  activeIndex: number;
+  colorVisionType: string;
+  data: LegendData[];
+  label: ReactNode;
+  setActivatorWidth: (width: number) => void;
+  theme?: string;
+  lastVisibleIndex?: number;
+  dimensions?: BoundingRect;
+}
+
+export const LEGEND_TOOLTIP_ID = 'legend-toolip';
+
+export function HiddenLegendTooltip({
+  activeIndex,
+  colorVisionType,
+  data,
+  theme,
+  label,
+  lastVisibleIndex = 0,
+  setActivatorWidth,
+  dimensions,
+}: Props) {
+  const selectedTheme = useTheme();
+  const {isFirefox} = useBrowserCheck();
+  const {id} = useChartContext();
+  const tooltipId = `${LEGEND_TOOLTIP_ID}_${id}`;
+  const container = useRootContainer(tooltipId);
+  const tooltipRef = useRef<HTMLDivElement>(null);
+  const activatorRef = useRef<HTMLButtonElement>(null);
+  useColorVisionEvents({enabled: true, root: LEGEND_TOOLTIP_ID, dimensions});
+
+  const defaultPosition = useMemo(
+    () => ({
+      top: 0,
+      left: 0,
+    }),
+    [],
+  );
```

Comment:
I think `TooltipAnimatedContainer` uses the css `transform` property since its more [performant to animate translations than positions](https://paulirish.com/2012/why-moving-elements-with-translate-is-better-than-posabs-topleft/). Since my tooltip is not animated, I just used css `position` properties instead.